spec/index.html

```diff
@@ -168,7 +168,7 @@ <h3>N3 document</h3>
           An <a>N3 graph</a> can have many more elements &mdash;
           <a>cited formulae</a> for quoting sets of statements,
           <a href="#logimpl">logical implications</a> for If-Then style, rule-based reasoning,
-          <a href="#variables">quantifications</a> for variables, and so on.
+          quantifications for <a>variables</a>, and so on.
```

Comment:
```suggestion
<a>lists</a>, and so on.
```
(we're no longer explicitly quantifying variables)

Reply:
This line came from @gkellogg; I just moved the `<a>` highlight.

Note that this is not the only occurrence of either `quantification` or `quantifier`, so this change should *probably* be one of may in another PR.

That said, I'm OK with the suggested change, if @gkellogg is.

Reply:
I am, but @doerthe @william-vw may feel differently.